Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -132,6 +132,16 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     }
     UIActivityViewController *shareViewController = [[UIActivityViewController alloc] initWithActivityItems:items applicationActivities:nil];
     
+    if ([shareViewController respondsToSelector:@selector(completionWithItemsHandler)]) {
+        shareViewController.completionWithItemsHandler = ^(NSString *activityType, BOOL completed, NSArray *returnedItems, NSError *activityError) {
```

Comment:
It should. it will work on iOS6, but developer should use iOS SDK 8 or higher in order to compile the code (otherwise Xcode won't recognize `completionWithItemsHandler` SDK method.
New completion handler format consists of simple types so it will work in any case.